hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/security/TestUGILoginFromKeytab.java

```diff
@@ -121,6 +143,9 @@ public void testUGILoginFromKeytab() throws Exception {
     final long firstLogin = user.getLastLogin();
     final LoginContext login1 = user.getLogin();
     Assert.assertNotNull(login1);
+    
+    // Sleep for 1 sec to have a difference between first and second login
+    Thread.sleep(1000);
```

Comment:
if you really want >1s of interval, make it a 2s, delay. There's too much risk of clock/sleep granularity issues

Reply:
Makes sense, changed it to 2 sec